java/samples/src/main/java/RocksDBSample.java

```diff
@@ -31,6 +31,7 @@ public static void main(final String[] args) {
          final Filter bloomFilter = new BloomFilter(10);
          final ReadOptions readOptions = new ReadOptions()
              .setFillCache(false);
+         final Statistics stats = new Statistics();
```

Comment:
Sorry you are of course correct. I did not see it initially in the GitHub diff view. Sorry.